Lifestyle/librelinkup.1m.py

```diff
@@ -3,19 +3,30 @@
 # for homebrew installation add '-S PATH="${PATH}:/opt/homebrew/bin/python3"' between /env and python3
 
 # <xbar.title>LibreLinkUp Status</xbar.title>
-# <xbar.version>v1.2</xbar.version>
-# <xbar.author>Florian Schlund</xbar.author>
-# <xbar.author.github>FloSchl8</xbar.author.github>
-# <xbar.desc>Gets your GCM from LibreLinkUp: https://librelinkup.com/</xbar.desc>
-# <xbar.dependencies>python3, python requests</xbar.dependencies>
-# <xbar.image>https://i.imgur.com/XUyOT9o.png</xbar.image>
-# <xbar.var>string(VAR_MAIL=""): Your e-mail.</xbar.var>
-# <xbar.var>string(VAR_PASSWORD=""): Your password.</xbar.var>
+# <xbar.version>v2.0</xbar.version>
+# <xbar.author>Maurici Abad</xbar.author>
+# <xbar.author.github>mauriciabad</xbar.author.github>
```

Comment:
We try to keep the original author in the metadata along with any contributors.

```suggestion
# <xbar.version>v2.0</xbar.version>
# <xbar.author>Florian Schlund,Maurici Abad</xbar.author>
# <xbar.author.github>FloSchl8,mauriciabad</xbar.author.github>
```

Reply:
Cool, I wasn't sure how to keep both of us :D